GerritDownloader/src/main/java/com/holmsted/gerrit/CommandLineParser.java

```diff
@@ -110,6 +115,14 @@ public String getServerName() {
     }
 
     @Nullable
+    public String getPrivateKey() {
+        if (privateKey != null) {
+          return privateKey;
+        }
+        return "~/.ssh/id_rsa";
```

Comment:
Did you test this? I wonder if ~ will work out of the box. You might have to expand it, see e.g. http://stackoverflow.com/questions/7163364/how-to-handle-in-file-paths